Source/WebCore/platform/graphics/GlyphPage.h

```diff
@@ -55,7 +56,7 @@ struct GlyphData {
 
     Glyph glyph;
     ColorGlyphType colorGlyphType;
-    const Font* font;
+    WeakPtr<const Font> font;
```

Comment:
Why are we using WeakPtr here?

Reply:
Ok, now using CheckedPtr more consistently.